google/cloud/security/scanner/audit/org_rules_engine.py

```diff
@@ -300,33 +323,32 @@ def find_violations(self, resource, policy_binding):
         """
         violations = []
```

Comment:
Same comment as earlier on making this a chain, and then returning it as a flattened list at the end.